logfire/_internal/cli.py

```diff
@@ -52,8 +51,9 @@ def parse_whoami(args: argparse.Namespace) -> None:
     """Show user authenticated username and the URL to your Logfire project."""
     data_dir = Path(args.data_dir)
     param_manager = ParamManager.create(data_dir)
-    base_url = param_manager.load_param('base_url', args.logfire_url)
+    base_url = param_manager.load_param('base_url', _get_logfire_url(args.logfire_url, args.region))
```

Comment:
Not for this PR, but load_param should probably be used for all commands, so that the env var `LOGFIRE_BASE_URL` always works.

Reply:
Same for region.